### PR TITLE
{lib}[system/system] cuSPARSELt v0.6.0.6 w/ CUDA 12.1.1

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera-skylake-build - Linux Rocky Linux 8.9, x86_64, Intel Xeon Processor (Skylake, IBRS, no TSX), Python 3.6.8
See https://gist.github.com/Micket/3cb8247289a588a1cf7d8502d34f35a5 for a full test report.

**Comment by Micket** [comment]
@boegelbot please test @ jsc-zen3

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Review by casparvl — APPROVED** [review]
Straightforward update, Lgtm!

**Comment by casparvl** [comment]
Going in, thanks @Micket!

File: easybuild/easyconfigs/c/cuSPARSELt/cuSPARSELt-0.6.0.6-CUDA-12.1.1.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+version = '0.6.0.6'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://docs.nvidia.com/cuda/cusparselt/index.html'
+description = """NVIDIA cuSPARSELt is a high-performance CUDA library dedicated to general matrix-matrix operations in
+which at least one operand is a sparse matrix"""
+
+toolchain = SYSTEM
+
+local_arch = {'arm64': 'sbsa'}.get(ARCH, ARCH)
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/c/cuSPARSELt/cuSPARSELt-0.6.0.6-CUDA-12.1.1.eb:13]: @Micket for me on Grace-Hopper what works here is `local_arch = {'aarch64': 'sbsa'}.get(ARCH, ARCH)`, not sure if it should be replaced or if we should check for both `arm64` and `aarch64`

File: easybuild/easyconfigs/c/cuSPARSELt/cuSPARSELt-0.6.0.6-CUDA-12.1.1.eb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+source_urls = ['https://developer.download.nvidia.com/compute/cusparselt/redist/libcusparse_lt/linux-%s/' % local_arch]
+sources = ['libcusparse_lt-linux-%s-%%(version)s-archive.tar.xz' % local_arch]
+checksums = [{
+    'libcusparse_lt-linux-x86_64-%(version)s-archive.tar.xz':
+        'da20d7a6d7a6674cb6e991439eaacd4661631c71a2f1026c41fba115c5c9c8b5',
+    'libcusparse_lt-linux-sbsa-%(version)s-archive.tar.xz':
+        '46b258c2c333f7324e03bdf6b13576be6153b1218fb531025631650e36515f45',
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/c/cuSPARSELt/cuSPARSELt-0.6.0.6-CUDA-12.1.1.eb:20]: @Micket I think this is the checksum of `libcusparse_lt-linux-x86_64-0.3.0.3-archive.tar.xz`

checksum of `ibcusparse_lt-linux-sbsa-0.6.0.6-archive.tar.xz` is `51db8182444b4e2c125397ea95e4996c022ef791a87ce7a7389204e77617297f`

Comment (Micket) [reply easybuild/easyconfigs/c/cuSPARSELt/cuSPARSELt-0.6.0.6-CUDA-12.1.1.eb:20]: Fixed in #20216

**Pull request merged**